ibis/backends/tests/test_register.py

```diff
@@ -577,14 +582,22 @@ def num_diamonds(data_dir):
     "in_table_name",
     [param(None, id="default"), param("fancy_stones", id="file_name")],
 )
-@pytest.mark.notyet(
-    ["flink", "impala", "mssql", "mysql", "postgres", "risingwave", "sqlite", "trino"]
-)
+@pytest.mark.notyet(["flink"])
+@pytest.mark.notimpl(["druid", "exasol", "oracle"])
 def test_read_csv(con, data_dir, in_table_name, num_diamonds):
     fname = "diamonds.csv"
     with pushd(data_dir / "csv"):
-        if con.name == "pyspark":
-            # pyspark doesn't respect CWD
+        if con.name in (
+            "pyspark",
+            "sqlite",
+            "mysql",
+            "postgres",
+            "risingwave",
+            "impala",
+            "mssql",
+            "trino",
+        ):
```

Comment:
This is a bit weird: for most of the backends in this list the notion of "current working directory" doesn't make sense.

Reply:
Yes, I am not sure what happened, I will double check it.

Reply:
It is `pyarrow` having trouble respecting CWD when trying to read the CSV file. I have to provide pyarrow with the full path to the csv file. 

does this make sense to you @cpcloud 

```
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
ibis/backends/__init__.py:1312: in read_csv
    file_system, path = fs.FileSystem.from_uri(path)
pyarrow/_fs.pyx:477: in pyarrow._fs.FileSystem.from_uri
    ???
pyarrow/error.pxi:155: in pyarrow.lib.pyarrow_internal_check_status
    ???
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

>   ???
E   pyarrow.lib.ArrowInvalid: URI has empty scheme: 'diamonds.csv'
```